VariablesHomework/VariablesHomework.cs

```diff
@@ -60,86 +73,86 @@ public void Operators()
             h--;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 24/*Answer here*/); // Make this pass
             ++h;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 24/*Answer here*/); // Make this pass
             --h;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 23/*Answer here*/); // Make this pass
             h += 1;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 24/*Answer here*/); // Make this pass
             h -= 1;
-            Assert.IsTrue(h == /*Answer here*/); // Make this pass
+            Assert.IsTrue(h == 23/*Answer here*/); // Make this pass
 
             string l = "Hello! ";
             string m = l + "You!";
-            Assert.IsTrue(m == /*Answer here*/); // Make this pass
+            Assert.IsTrue(m == "Hello!You!"/*Answer here*/); // Make this pass
 
             int v = 5;
             int x = v % 4;
-            Assert.IsTrue(x == /*Answer here*/); // Make this pass
+            Assert.IsTrue(x == 1/*Answer here*/); // Make this pass
 
             int finalInt = 15;
             int divisionResult = 15 / 3;
-            Assert.IsTrue(divisionResult == /*Answer here*/); // Make this pass
+            Assert.IsTrue(divisionResult == 5/*Answer here*/); // Make this pass
 
             finalInt = 15;
             finalInt /= 3;
-            Assert.IsTrue(finalInt == /*Answer here*/); // Make this pass
+            Assert.IsTrue(finalInt == 5/*Answer here*/); // Make this pass
 
             int orderOfOperations = 2;
             orderOfOperations = 2 + 12 * (orderOfOperations * 3);
-            Assert.IsTrue(orderOfOperations == /*Answer here*/); // Make this pass
+            Assert.IsTrue(orderOfOperations == 74/*Answer here*/); // Make this pass
         }
 
         [TestMethod]
         public void Ranges()
         {
-            Assert.IsTrue(int.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(int.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(float.MinValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MaxValue == /*Answer here*/); // Make this pass
-            Assert.IsTrue(double.MinValue == /*Answer here*/); // Make this pass
+            Assert.IsTrue(int.MaxValue == 2147483647/*Answer here*/); // Make this pass
+            Assert.IsTrue(int.MinValue == -2147483648/*Answer here*/); // Make this pass
+            Assert.IsTrue(float.MaxValue == 3.40282347E+38f/*Answer here*/); // Make this pass
+            Assert.IsTrue(float.MinValue == -3.40282347E+38f/*Answer here*/); // Make this pass
+            Assert.IsTrue(double.MaxValue == 1.7E308/*Answer here*/); // Make this pass
```

Comment:
This answer looks like it was pulled from documentation (which isn't a bad thing), but it's not exact. The test doesn't pass.